scripts/set_gem_timing

```diff
@@ -8,54 +8,54 @@ IOC_PATH_SC="ioc/fee/GasDetDAQ/R4.0.22-NOTS"
 IOC_PATH=0
 
 # User input validation
-if [[ -n $1 ]]; then # if the user passed at least one arg
-  if [[ $1 != "$NC_STRING" ]] && [[ $1 != "$SC_STRING" ]]; then #validate argument is either SC or NC
-    echo "Error. Argument must be either: [NC or SC]"
-    exit 1
-  else
-    echo "User passed argument of: $1"
-  fi
+if [[ -n $1 ]]; then                                              # if the user passed at least one arg
+    if [[ $1 != "$NC_STRING" ]] && [[ $1 != "$SC_STRING" ]]; then #validate argument is either SC or NC
+        echo "Error. Argument must be either: [NC or SC]"
+        exit 1
+    else
+        echo "User passed argument of: $1"
+    fi
 else
-  echo "Error. Must pass at least one argument"
-  exit 1
+    echo "Error. Must pass at least one argument"
+    exit 1
 fi
 
 if [[ -n $2 ]]; then
-  echo "Error. Must pass strictly one argument, at least one additional arg: $2, please run again with exclusively [NC or SC] as argument"
-  exit 1
+    echo "Error. Must pass strictly one argument, at least one additional arg: $2, please run again with exclusively [NC or SC] as argument"
+    exit 1
 fi
 
 #get caget and caput
 source /cds/group/pcds/setup/epicsenv-cur.sh #Note: (josh) we should only source the actual executables we are using, this is heavy handed. shellcheck does not love this; however if this path doesnt exist we have bigger problems
 #get imgr
-IMGR_CMD_PATH="/reg/g/pcds/engineering_tools/latest-released/scripts/imgr"
+IMGR_CMD_PATH="/cds/group/pcds/engineering_tools/latest-released/scripts/imgr"
```

Comment:
This is kind of bad behavior that should be addressed in a separate PR. I'm accumulating issues for the repo as I am ending up reading most of the scripts to double-check the reformatting here.

This should use the imgr script from the same release, not have a hard-coded link to whatever the latest release is.